src/toolbars/SelectionBar.cpp

```diff
@@ -60,39 +61,50 @@ with changes in the SelectionBar.
 #include "../ProjectSettings.h"
 #include "ViewInfo.h"
 #include "AllThemeResources.h"
+#include "ImageManipulation.h"
 #include "../widgets/AButton.h"
 #include "../widgets/auStaticText.h"
 #include "../widgets/BasicMenu.h"
 #include "../widgets/NumericTextCtrl.h"
 #include "../widgets/wxWidgetsWindowPlacement.h"
 
-#if wxUSE_ACCESSIBILITY
-#include "../widgets/WindowAccessible.h"
-#endif
+namespace
+{
+IntSetting sSelectionMode { "/SelectionToolbarMode", 0 };
```

Comment:
If you add a setting object, be sure to eliminate all repetitions of the path elsewhere.  So you must also change PluginMenus.cpp